Fix aquarius install instructions

While reviewing https://github.com/oceanprotocol/aquarius/pull/515, I recommended dropping the `python3-dev` dependency on the grounds that it wasn't needed (in favor of `python3`).

I now realize that it is needed because some of our other dependencies only exist as source packages and require the Python headers to build and install properly.

Also, it looks like we need to install `wheel` first.

`--no-cache-dir` was used to emulate a clean install.

<details>
<summary>Install aquarius console logs - error missing python headers</summary>
<p>

```python
david@david-Inspiron-5502:~/projects/aquarius$ apt list --installed python3-dev
Listing... Done
david@david-Inspiron-5502:~/projects/aquarius$ python3 -m venv venv
david@david-Inspiron-5502:~/projects/aquarius$ source venv/bin/activate
(venv) david@david-Inspiron-5502:~/projects/aquarius$ pip install --no-cache-dir -r requirements.txt 
Obtaining file:///home/david/projects/aquarius (from -r requirements.txt (line 12))
Collecting Flask-Cors==3.0.10
  Downloading Flask_Cors-3.0.10-py2.py3-none-any.whl (14 kB)
Collecting Flask==2.0.1
  Downloading Flask-2.0.1-py3-none-any.whl (94 kB)
     |████████████████████████████████| 94 kB 5.7 MB/s 
Collecting Jinja2>=2.10.1
  Downloading Jinja2-3.0.1-py3-none-any.whl (133 kB)
     |████████████████████████████████| 133 kB 37.3 MB/s 
Collecting PyYAML==5.4.1
  Downloading PyYAML-5.4.1-cp38-cp38-manylinux1_x86_64.whl (662 kB)
     |████████████████████████████████| 662 kB 78.5 MB/s 
Collecting coloredlogs==15.0.1
  Downloading coloredlogs-15.0.1-py2.py3-none-any.whl (46 kB)
     |████████████████████████████████| 46 kB 972 kB/s 
Collecting eciespy
  Downloading eciespy-0.3.10-py3-none-any.whl (10 kB)
Collecting flask-swagger-ui==3.36.0
  Downloading flask-swagger-ui-3.36.0.tar.gz (4.9 MB)
     |████████████████████████████████| 4.9 MB 46.2 MB/s 
Collecting flask-swagger==0.2.14
  Downloading flask-swagger-0.2.14.tar.gz (8.6 kB)
Collecting gevent
  Downloading gevent-21.1.2-cp38-cp38-manylinux2010_x86_64.whl (6.3 MB)
     |████████████████████████████████| 6.3 MB 103.9 MB/s 
Collecting gunicorn==20.1.0
  Downloading gunicorn-20.1.0-py3-none-any.whl (79 kB)
     |████████████████████████████████| 79 kB 89.5 MB/s 
Collecting json-sempai==0.4.0
  Downloading json_sempai-0.4.0-py2.py3-none-any.whl (6.2 kB)
Collecting jsonschema==3.2.0
  Downloading jsonschema-3.2.0-py2.py3-none-any.whl (56 kB)
     |████████████████████████████████| 56 kB 35.1 MB/s 
Collecting ocean-contracts==0.6.4
  Downloading ocean_contracts-0.6.4-py2.py3-none-any.whl (675 kB)
     |████████████████████████████████| 675 kB 40.4 MB/s 
Collecting oceandb-driver-interface==0.2.0
  Downloading oceandb_driver_interface-0.2.0-py2.py3-none-any.whl (11 kB)
Collecting oceandb-elasticsearch-driver==0.4.4
  Downloading oceandb_elasticsearch_driver-0.4.4-py2.py3-none-any.whl (15 kB)
Collecting python-dateutil==2.8.1
  Downloading python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
     |████████████████████████████████| 227 kB 25.4 MB/s 
Collecting pytz==2021.1
  Downloading pytz-2021.1-py2.py3-none-any.whl (510 kB)
     |████████████████████████████████| 510 kB 74.9 MB/s 
Collecting requests>=2.21.0
  Downloading requests-2.26.0-py2.py3-none-any.whl (62 kB)
     |████████████████████████████████| 62 kB 28.9 MB/s 
Collecting web3==5.19.0
  Downloading web3-5.19.0-py3-none-any.whl (470 kB)
     |████████████████████████████████| 470 kB 45.8 MB/s 
Collecting black
  Downloading black-21.7b0-py3-none-any.whl (141 kB)
     |████████████████████████████████| 141 kB 36.4 MB/s 
Collecting bumpversion==0.6.0
  Downloading bumpversion-0.6.0-py2.py3-none-any.whl (8.4 kB)
Collecting codacy-coverage==1.3.11
  Downloading codacy_coverage-1.3.11-py2.py3-none-any.whl (7.5 kB)
Collecting coverage==5.5
  Downloading coverage-5.5-cp38-cp38-manylinux2010_x86_64.whl (245 kB)
     |████████████████████████████████| 245 kB 28.5 MB/s 
Collecting flake8
  Downloading flake8-3.9.2-py2.py3-none-any.whl (73 kB)
     |████████████████████████████████| 73 kB 38.3 MB/s 
Collecting freezegun==1.1.0
  Downloading freezegun-1.1.0-py2.py3-none-any.whl (16 kB)
Collecting isort
  Downloading isort-5.9.2-py3-none-any.whl (105 kB)
     |████████████████████████████████| 105 kB 40.7 MB/s 
Collecting licenseheaders
  Downloading licenseheaders-0.8.8-py3-none-any.whl (21 kB)
Collecting mccabe==0.6.1
  Downloading mccabe-0.6.1-py2.py3-none-any.whl (8.6 kB)
Collecting pkginfo==1.7.1
  Downloading pkginfo-1.7.1-py2.py3-none-any.whl (25 kB)
Collecting pre-commit
  Downloading pre_commit-2.13.0-py2.py3-none-any.whl (190 kB)
     |████████████████████████████████| 190 kB 38.1 MB/s 
Collecting pylint==2.9.3
  Downloading pylint-2.9.3-py3-none-any.whl (372 kB)
     |████████████████████████████████| 372 kB 54.5 MB/s 
Collecting pytest
  Downloading pytest-6.2.4-py3-none-any.whl (280 kB)
     |████████████████████████████████| 280 kB 74.3 MB/s 
Collecting pytest-env
  Downloading pytest-env-0.6.2.tar.gz (1.7 kB)
Collecting tox
  Downloading tox-3.24.0-py2.py3-none-any.whl (85 kB)
     |████████████████████████████████| 85 kB 79.0 MB/s 
Collecting twine==3.4.1
  Downloading twine-3.4.1-py3-none-any.whl (34 kB)
Collecting watchdog==2.1.3
  Downloading watchdog-2.1.3-py3-none-manylinux2014_x86_64.whl (75 kB)
     |████████████████████████████████| 75 kB 44.4 MB/s 
Collecting Six
  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting itsdangerous>=2.0
  Downloading itsdangerous-2.0.1-py3-none-any.whl (18 kB)
Collecting click>=7.1.2
  Downloading click-8.0.1-py3-none-any.whl (97 kB)
     |████████████████████████████████| 97 kB 70.1 MB/s 
Collecting Werkzeug>=2.0
  Downloading Werkzeug-2.0.1-py3-none-any.whl (288 kB)
     |████████████████████████████████| 288 kB 45.2 MB/s 
Collecting MarkupSafe>=2.0
  Downloading MarkupSafe-2.0.1-cp38-cp38-manylinux2010_x86_64.whl (30 kB)
Collecting humanfriendly>=9.1
  Downloading humanfriendly-9.2-py2.py3-none-any.whl (86 kB)
     |████████████████████████████████| 86 kB 2.5 MB/s 
Collecting pycryptodome<4.0.0,>=3.9.9
  Downloading pycryptodome-3.10.1-cp35-abi3-manylinux2010_x86_64.whl (1.9 MB)
     |████████████████████████████████| 1.9 MB 35.7 MB/s 
Collecting eth-keys<0.4.0,>=0.3.3
  Downloading eth_keys-0.3.3-py3-none-any.whl (20 kB)
Collecting coincurve<16,>=13
  Downloading coincurve-15.0.0-cp38-cp38-manylinux2014_x86_64.whl (556 kB)
     |████████████████████████████████| 556 kB 76.1 MB/s 
Collecting zope.interface
  Downloading zope.interface-5.4.0-cp38-cp38-manylinux2010_x86_64.whl (259 kB)
     |████████████████████████████████| 259 kB 81.4 MB/s 
Requirement already satisfied: setuptools in ./venv/lib/python3.8/site-packages (from gevent->ocean-aquarius==2.2.12->-r requirements.txt (line 12)) (44.0.0)
Collecting zope.event
  Downloading zope.event-4.5.0-py2.py3-none-any.whl (6.8 kB)
Collecting greenlet<2.0,>=0.4.17; platform_python_implementation == "CPython"
  Downloading greenlet-1.1.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (164 kB)
     |████████████████████████████████| 164 kB 83.5 MB/s 
Collecting pyrsistent>=0.14.0
  Downloading pyrsistent-0.18.0-cp38-cp38-manylinux1_x86_64.whl (118 kB)
     |████████████████████████████████| 118 kB 72.5 MB/s 
Collecting attrs>=17.4.0
  Downloading attrs-21.2.0-py2.py3-none-any.whl (53 kB)
     |████████████████████████████████| 53 kB 41.1 MB/s 
Collecting elasticsearch
  Downloading elasticsearch-7.13.3-py2.py3-none-any.whl (356 kB)
     |████████████████████████████████| 356 kB 79.0 MB/s 
Collecting urllib3<1.27,>=1.21.1
  Downloading urllib3-1.26.6-py2.py3-none-any.whl (138 kB)
     |████████████████████████████████| 138 kB 7.0 MB/s 
Collecting idna<4,>=2.5; python_version >= "3"
  Downloading idna-3.2-py3-none-any.whl (59 kB)
     |████████████████████████████████| 59 kB 23.7 MB/s 
Collecting charset-normalizer~=2.0.0; python_version >= "3"
  Downloading charset_normalizer-2.0.3-py3-none-any.whl (35 kB)
Collecting certifi>=2017.4.17
  Downloading certifi-2021.5.30-py2.py3-none-any.whl (145 kB)
     |████████████████████████████████| 145 kB 78.0 MB/s 
Collecting protobuf<4,>=3.10.0
  Downloading protobuf-3.17.3-cp38-cp38-manylinux_2_5_x86_64.manylinux1_x86_64.whl (1.0 MB)
     |████████████████████████████████| 1.0 MB 94.1 MB/s 
Collecting eth-typing<3.0.0,>=2.0.0
  Downloading eth_typing-2.2.2-py3-none-any.whl (6.2 kB)
Collecting eth-hash[pycryptodome]<1.0.0,>=0.2.0
  Downloading eth_hash-0.3.1-py3-none-any.whl (8.9 kB)
Collecting hexbytes<1.0.0,>=0.1.0
  Downloading hexbytes-0.2.1-py3-none-any.whl (6.0 kB)
Collecting ipfshttpclient==0.7.0a1
  Downloading ipfshttpclient-0.7.0a1-py3-none-any.whl (231 kB)
     |████████████████████████████████| 231 kB 92.9 MB/s 
Collecting eth-utils<2.0.0,>=1.9.5
  Downloading eth_utils-1.10.0-py3-none-any.whl (24 kB)
Collecting lru-dict<2.0.0,>=1.1.6
  Downloading lru-dict-1.1.7.tar.gz (10 kB)
Collecting eth-abi<3.0.0,>=2.0.0b6
  Downloading eth_abi-2.1.1-py3-none-any.whl (27 kB)
Collecting eth-account<0.6.0,>=0.5.3
  Downloading eth_account-0.5.4-py3-none-any.whl (94 kB)
     |████████████████████████████████| 94 kB 29.2 MB/s 
Collecting websockets<9.0.0,>=8.1.0
  Downloading websockets-8.1-cp38-cp38-manylinux2010_x86_64.whl (78 kB)
     |████████████████████████████████| 78 kB 98.3 MB/s 
Collecting mypy-extensions>=0.4.3
  Downloading mypy_extensions-0.4.3-py2.py3-none-any.whl (4.5 kB)
Collecting pathspec<1,>=0.8.1
  Downloading pathspec-0.9.0-py2.py3-none-any.whl (31 kB)
Collecting regex>=2020.1.8
  Downloading regex-2021.7.6-cp38-cp38-manylinux2014_x86_64.whl (737 kB)
     |████████████████████████████████| 737 kB 116.7 MB/s 
Collecting appdirs
  Downloading appdirs-1.4.4-py2.py3-none-any.whl (9.6 kB)
Collecting tomli<2.0.0,>=0.2.6
  Downloading tomli-1.0.4-py3-none-any.whl (11 kB)
Collecting bump2version
  Downloading bump2version-1.0.1-py2.py3-none-any.whl (22 kB)
Collecting pycodestyle<2.8.0,>=2.7.0
  Downloading pycodestyle-2.7.0-py2.py3-none-any.whl (41 kB)
     |████████████████████████████████| 41 kB 24.9 MB/s 
Collecting pyflakes<2.4.0,>=2.3.0
  Downloading pyflakes-2.3.1-py2.py3-none-any.whl (68 kB)
     |████████████████████████████████| 68 kB 47.9 MB/s 
Collecting nodeenv>=0.11.1
  Downloading nodeenv-1.6.0-py2.py3-none-any.whl (21 kB)
Collecting identify>=1.0.0
  Downloading identify-2.2.11-py2.py3-none-any.whl (98 kB)
     |████████████████████████████████| 98 kB 35.6 MB/s 
Collecting toml
  Downloading toml-0.10.2-py2.py3-none-any.whl (16 kB)
Collecting cfgv>=2.0.0
  Downloading cfgv-3.3.0-py2.py3-none-any.whl (7.3 kB)
Collecting virtualenv>=20.0.8
  Downloading virtualenv-20.6.0-py2.py3-none-any.whl (5.3 MB)
     |████████████████████████████████| 5.3 MB 32.0 MB/s 
Collecting astroid<2.7,>=2.6.2
  Downloading astroid-2.6.2-py3-none-any.whl (228 kB)
     |████████████████████████████████| 228 kB 79.1 MB/s 
Collecting py>=1.8.2
  Downloading py-1.10.0-py2.py3-none-any.whl (97 kB)
     |████████████████████████████████| 97 kB 68.7 MB/s 
Collecting packaging
  Downloading packaging-21.0-py3-none-any.whl (40 kB)
     |████████████████████████████████| 40 kB 39.5 MB/s 
Collecting iniconfig
  Downloading iniconfig-1.1.1-py2.py3-none-any.whl (5.0 kB)
Collecting pluggy<1.0.0a1,>=0.12
  Downloading pluggy-0.13.1-py2.py3-none-any.whl (18 kB)
Collecting filelock>=3.0.0
  Downloading filelock-3.0.12-py3-none-any.whl (7.6 kB)
Collecting colorama>=0.4.3
  Downloading colorama-0.4.4-py2.py3-none-any.whl (16 kB)
Collecting requests-toolbelt!=0.9.0,>=0.8.0
  Downloading requests_toolbelt-0.9.1-py2.py3-none-any.whl (54 kB)
     |████████████████████████████████| 54 kB 36.6 MB/s 
Collecting keyring>=15.1
  Downloading keyring-23.0.1-py3-none-any.whl (33 kB)
Collecting rfc3986>=1.4.0
  Downloading rfc3986-1.5.0-py2.py3-none-any.whl (31 kB)
Collecting tqdm>=4.14
  Downloading tqdm-4.61.2-py2.py3-none-any.whl (76 kB)
     |████████████████████████████████| 76 kB 58.2 MB/s 
Collecting importlib-metadata>=3.6
  Downloading importlib_metadata-4.6.1-py3-none-any.whl (17 kB)
Collecting readme-renderer>=21.0
  Downloading readme_renderer-29.0-py2.py3-none-any.whl (15 kB)
Collecting asn1crypto
  Downloading asn1crypto-1.4.0-py2.py3-none-any.whl (104 kB)
     |████████████████████████████████| 104 kB 56.4 MB/s 
Collecting cffi>=1.3.0
  Downloading cffi-1.14.6-cp38-cp38-manylinux1_x86_64.whl (411 kB)
     |████████████████████████████████| 411 kB 81.6 MB/s 
Collecting multiaddr>=0.0.7
  Downloading multiaddr-0.0.9-py2.py3-none-any.whl (16 kB)
Collecting cytoolz<1.0.0,>=0.10.1; implementation_name == "cpython"
  Downloading cytoolz-0.11.0.tar.gz (477 kB)
     |████████████████████████████████| 477 kB 81.5 MB/s 
Collecting parsimonious<0.9.0,>=0.8.0
  Downloading parsimonious-0.8.1.tar.gz (45 kB)
     |████████████████████████████████| 45 kB 43.3 MB/s 
Collecting rlp<3,>=1.0.0
  Downloading rlp-2.0.1-py2.py3-none-any.whl (20 kB)
Collecting eth-keyfile<0.6.0,>=0.5.0
  Downloading eth_keyfile-0.5.1-py3-none-any.whl (8.3 kB)
Collecting eth-rlp<2,>=0.1.2
  Downloading eth_rlp-0.2.1-py3-none-any.whl (5.0 kB)
Collecting bitarray<1.3.0,>=1.2.1
  Downloading bitarray-1.2.2.tar.gz (48 kB)
     |████████████████████████████████| 48 kB 36.8 MB/s 
Collecting backports.entry-points-selectable>=1.0.4
  Downloading backports.entry_points_selectable-1.1.0-py2.py3-none-any.whl (6.2 kB)
Collecting platformdirs<3,>=2
  Downloading platformdirs-2.0.2-py2.py3-none-any.whl (10 kB)
Collecting distlib<1,>=0.3.1
  Downloading distlib-0.3.2-py2.py3-none-any.whl (338 kB)
     |████████████████████████████████| 338 kB 61.6 MB/s 
Collecting wrapt<1.13,>=1.11
  Downloading wrapt-1.12.1.tar.gz (27 kB)
Collecting lazy-object-proxy>=1.4.0
  Downloading lazy_object_proxy-1.6.0-cp38-cp38-manylinux1_x86_64.whl (58 kB)
     |████████████████████████████████| 58 kB 69.6 MB/s 
Collecting pyparsing>=2.0.2
  Downloading pyparsing-2.4.7-py2.py3-none-any.whl (67 kB)
     |████████████████████████████████| 67 kB 79.7 MB/s 
Collecting SecretStorage>=3.2; sys_platform == "linux"
  Downloading SecretStorage-3.3.1-py3-none-any.whl (15 kB)
Collecting jeepney>=0.4.2; sys_platform == "linux"
  Downloading jeepney-0.7.0-py3-none-any.whl (53 kB)
     |████████████████████████████████| 53 kB 46.7 MB/s 
Collecting zipp>=0.5
  Downloading zipp-3.5.0-py3-none-any.whl (5.7 kB)
Collecting bleach>=2.1.0
  Downloading bleach-3.3.1-py2.py3-none-any.whl (146 kB)
     |████████████████████████████████| 146 kB 32.1 MB/s 
Collecting Pygments>=2.5.1
  Downloading Pygments-2.9.0-py3-none-any.whl (1.0 MB)
     |████████████████████████████████| 1.0 MB 36.4 MB/s 
Collecting docutils>=0.13.1
  Downloading docutils-0.17.1-py2.py3-none-any.whl (575 kB)
     |████████████████████████████████| 575 kB 101.8 MB/s 
Collecting pycparser
  Downloading pycparser-2.20-py2.py3-none-any.whl (112 kB)
     |████████████████████████████████| 112 kB 56.5 MB/s 
Collecting varint
  Downloading varint-1.0.2.tar.gz (1.9 kB)
Collecting netaddr
  Downloading netaddr-0.8.0-py2.py3-none-any.whl (1.9 MB)
     |████████████████████████████████| 1.9 MB 49.8 MB/s 
Collecting base58
  Downloading base58-2.1.0-py3-none-any.whl (5.6 kB)
Collecting toolz>=0.8.0
  Downloading toolz-0.11.1-py3-none-any.whl (55 kB)
     |████████████████████████████████| 55 kB 33.6 MB/s 
Collecting cryptography>=2.0
  Downloading cryptography-3.4.7-cp36-abi3-manylinux2014_x86_64.whl (3.2 MB)
     |████████████████████████████████| 3.2 MB 66.4 MB/s 
Collecting webencodings
  Downloading webencodings-0.5.1-py2.py3-none-any.whl (11 kB)
Building wheels for collected packages: flask-swagger-ui, flask-swagger, pytest-env, lru-dict, cytoolz, parsimonious, bitarray, wrapt, varint
  Building wheel for flask-swagger-ui (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/david/projects/aquarius/venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ppp_zq7v/flask-swagger-ui/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ppp_zq7v/flask-swagger-ui/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-fgmclc3u
       cwd: /tmp/pip-install-ppp_zq7v/flask-swagger-ui/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for flask-swagger-ui
  Running setup.py clean for flask-swagger-ui
  Building wheel for flask-swagger (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/david/projects/aquarius/venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ppp_zq7v/flask-swagger/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ppp_zq7v/flask-swagger/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-3xarpykz
       cwd: /tmp/pip-install-ppp_zq7v/flask-swagger/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for flask-swagger
  Running setup.py clean for flask-swagger
  Building wheel for pytest-env (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/david/projects/aquarius/venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ppp_zq7v/pytest-env/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ppp_zq7v/pytest-env/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-q2v1xn4g
       cwd: /tmp/pip-install-ppp_zq7v/pytest-env/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for pytest-env
  Running setup.py clean for pytest-env
  Building wheel for lru-dict (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/david/projects/aquarius/venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ppp_zq7v/lru-dict/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ppp_zq7v/lru-dict/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-6vqd2ico
       cwd: /tmp/pip-install-ppp_zq7v/lru-dict/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for lru-dict
  Running setup.py clean for lru-dict
  Building wheel for cytoolz (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/david/projects/aquarius/venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ppp_zq7v/cytoolz/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ppp_zq7v/cytoolz/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-ohof8ybi
       cwd: /tmp/pip-install-ppp_zq7v/cytoolz/
  Complete output (7 lines):
  ALERT: Cython not installed.  Building without Cython.
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for cytoolz
  Running setup.py clean for cytoolz
  Building wheel for parsimonious (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/david/projects/aquarius/venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ppp_zq7v/parsimonious/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ppp_zq7v/parsimonious/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-fwfi74rm
       cwd: /tmp/pip-install-ppp_zq7v/parsimonious/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for parsimonious
  Running setup.py clean for parsimonious
  Building wheel for bitarray (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/david/projects/aquarius/venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ppp_zq7v/bitarray/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ppp_zq7v/bitarray/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-2_qbdt7l
       cwd: /tmp/pip-install-ppp_zq7v/bitarray/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for bitarray
  Running setup.py clean for bitarray
  Building wheel for wrapt (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/david/projects/aquarius/venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ppp_zq7v/wrapt/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ppp_zq7v/wrapt/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-cx9xst4l
       cwd: /tmp/pip-install-ppp_zq7v/wrapt/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for wrapt
  Running setup.py clean for wrapt
  Building wheel for varint (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/david/projects/aquarius/venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ppp_zq7v/varint/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ppp_zq7v/varint/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-2yhodk73
       cwd: /tmp/pip-install-ppp_zq7v/varint/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for varint
  Running setup.py clean for varint
Failed to build flask-swagger-ui flask-swagger pytest-env lru-dict cytoolz parsimonious bitarray wrapt varint
Installing collected packages: Six, itsdangerous, MarkupSafe, Jinja2, click, Werkzeug, Flask, Flask-Cors, PyYAML, humanfriendly, coloredlogs, pycryptodome, eth-typing, toolz, cytoolz, eth-hash, eth-utils, eth-keys, asn1crypto, pycparser, cffi, coincurve, eciespy, flask-swagger-ui, flask-swagger, zope.interface, zope.event, greenlet, gevent, gunicorn, json-sempai, pyrsistent, attrs, jsonschema, ocean-contracts, oceandb-driver-interface, certifi, urllib3, elasticsearch, oceandb-elasticsearch-driver, python-dateutil, pytz, idna, charset-normalizer, requests, protobuf, hexbytes, varint, netaddr, base58, multiaddr, ipfshttpclient, lru-dict, parsimonious, eth-abi, rlp, eth-keyfile, eth-rlp, bitarray, eth-account, websockets, web3, mypy-extensions, pathspec, regex, appdirs, tomli, black, bump2version, bumpversion, codacy-coverage, coverage, mccabe, pycodestyle, pyflakes, flake8, freezegun, isort, licenseheaders, pkginfo, nodeenv, identify, toml, cfgv, filelock, backports.entry-points-selectable, platformdirs, distlib, virtualenv, pre-commit, wrapt, lazy-object-proxy, astroid, pylint, py, pyparsing, packaging, iniconfig, pluggy, pytest, pytest-env, tox, colorama, requests-toolbelt, zipp, importlib-metadata, cryptography, jeepney, SecretStorage, keyring, rfc3986, tqdm, webencodings, bleach, Pygments, docutils, readme-renderer, twine, watchdog, ocean-aquarius
    Running setup.py install for cytoolz ... error
    ERROR: Command errored out with exit status 1:
     command: /home/david/projects/aquarius/venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ppp_zq7v/cytoolz/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ppp_zq7v/cytoolz/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-l0gi9sxz/install-record.txt --single-version-externally-managed --compile --install-headers /home/david/projects/aquarius/venv/include/site/python3.8/cytoolz
         cwd: /tmp/pip-install-ppp_zq7v/cytoolz/
    Complete output (57 lines):
    ALERT: Cython not installed.  Building without Cython.
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.8
    creating build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/__init__.py -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/_signatures.py -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/utils_test.py -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/_version.py -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/compatibility.py -> build/lib.linux-x86_64-3.8/cytoolz
    creating build/lib.linux-x86_64-3.8/cytoolz/curried
    copying cytoolz/curried/__init__.py -> build/lib.linux-x86_64-3.8/cytoolz/curried
    copying cytoolz/curried/exceptions.py -> build/lib.linux-x86_64-3.8/cytoolz/curried
    copying cytoolz/curried/operator.py -> build/lib.linux-x86_64-3.8/cytoolz/curried
    copying cytoolz/recipes.pyx -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/dicttoolz.pyx -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/functoolz.pyx -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/utils.pyx -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/itertoolz.pyx -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/cpython.pxd -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/utils.pxd -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/itertoolz.pxd -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/__init__.pxd -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/functoolz.pxd -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/dicttoolz.pxd -> build/lib.linux-x86_64-3.8/cytoolz
    copying cytoolz/recipes.pxd -> build/lib.linux-x86_64-3.8/cytoolz
    creating build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_inspect_args.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_utils.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_signatures.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_dev_skip_test.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_curried_toolzlike.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_compatibility.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_itertoolz.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_none_safe.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_functoolz.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_serialization.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/dev_skip_test.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_embedded_sigs.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_dicttoolz.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_curried.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_doctests.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_recipes.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_docstrings.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    copying cytoolz/tests/test_tlz.py -> build/lib.linux-x86_64-3.8/cytoolz/tests
    running build_ext
    building 'cytoolz.dicttoolz' extension
    creating build/temp.linux-x86_64-3.8
    creating build/temp.linux-x86_64-3.8/cytoolz
    x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/home/david/projects/aquarius/venv/include -I/usr/include/python3.8 -c cytoolz/dicttoolz.c -o build/temp.linux-x86_64-3.8/cytoolz/dicttoolz.o
    cytoolz/dicttoolz.c:17:10: fatal error: Python.h: No such file or directory
       17 | #include "Python.h"
          |          ^~~~~~~~~~
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
    ----------------------------------------
ERROR: Command errored out with exit status 1: /home/david/projects/aquarius/venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ppp_zq7v/cytoolz/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ppp_zq7v/cytoolz/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-l0gi9sxz/install-record.txt --single-version-externally-managed --compile --install-headers /home/david/projects/aquarius/venv/include/site/python3.8/cytoolz Check the logs for full command output.
(venv) david@david-Inspiron-5502:~/projects/aquarius$ 
```

</p>
</details>

--- a/README.md
+++ b/README.md
@@ -161,7 +161,7 @@ Install Aquarius's OS-level requirements:
 
 ```bash
 sudo apt update
-sudo apt install python3
+sudo apt install python3-dev
 ```
 
 It is recommended that you create and activate a virtual environment in order to install the dependencies.
@@ -174,6 +174,7 @@ source venv/bin/activate
 At this point, with the Elasticsearch database already running, now you can start the Aquarius server:
 
 ```bash
+pip install wheel
 pip install -r requirements.txt
 export FLASK_APP=aquarius/run.py
 export CONFIG_FILE=config.ini

